docs/guides/custom-domains/index.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Custom Domains
+---
+
+## Overview
+[myzrok.io](https://myzrok.io) is a hosted zrok-as-a-service offering that provides a way for you bring a custom DNS name for zrok shares.
+For example, let's say you own the domain `foo.example.io`, you can leverage zrok custom domains to
+create ephemeral shares such as: `https://vw8jbg4ijz5g.foo.example.io`
+or [reserved shares](/concepts/sharing-reserved.md) such as `https://myshare.foo.example.io`.
+
+Custom domains require a Pro subscription with [myzrok.io](https://myzrok.io).
+If you don't already have an account, you can sign up for one [here](https://myzrok.io).
+
+[myzrok.io](https://myzrok.io) provides a guided setup with just a few easy steps!
+
+1. Bring your own custom domain name
+2. Create DNS validation records and a CNAME record
+3. Wait for zrok to validate your records and finalize configuration
+4. Start sharing!
+
+Detailed setup instructions are documented below.
+
+### Prerequisites
+:::note
+In order to create a custom domain in zrok, **you must already own the domain you want to use.**
+:::
+
+During the setup process you will need to create DNS records to validate ownership and to allow a certiicate to be issued
+on behalf of your domain. Once you have your domain registered, you can begin the process of setting up your custom
+domain with zrok!
+
+### Create Your Custom Domain
+
+Log into the myzrok Console and access the domains page by clicking on the globe icon in the left navigation menu.
+
+![myzrok_domains_page](images/myzrok_domains_page.png)
+
+Custom domains require a Pro subscription in [myzrok.io](https://myzrok.io)
+
+Click the CREATE button on the top right of the page to get started.
+When you click the create button you’ll be presented with a form to allow you to enable your custom domain.
+Enter your domain into the form field and click CREATE. This will begin the process for setting up your custom domain.
+A new managed TLS certificate will be created to host traffic on your domain's behalf.
+
+![myzrok_add_domain](images/myzrok_add_domain.png)
+
+This may take a few minutes.
+You may close the form at this time and come back when your domain is *pending validation.*
+Once your certificate is ready, you’ll be presented with instructions on how to set up your DNS records.
```

Comment:
A rough estimate for timing will be appreciated. At a minimum, how long to wait before raising a support request.

Reply:
We don't really have control over this, and AWS doesn't give estimates on it either